app/src/main/kotlin/com/intive/picover/common/di/AppModule.kt

```diff
@@ -0,0 +1,22 @@
+package com.intive.picover.common.di
+
+import com.google.firebase.storage.FirebaseStorage
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+import kotlinx.coroutines.Dispatchers
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AppModule {
+
+	@Provides
+	@Singleton
```

Comment:
Look here: https://dagger.dev/hilt/components.html section "When to scope?":
"Scoping a binding has a cost on both the generated code size and its runtime performance so use scoping sparingly. The general rule for determining if a binding should be scoped is to only scope the binding if it’s required for the correctness of the code. If you think a binding should be scoped for purely performance reasons, first verify that the performance is an issue, and if it is consider using @Reusable instead of a component scope."

Reply:
Thank you. Please see the update.